package.json

```diff
@@ -63,6 +63,15 @@
     "collectCoverage": true,
     "coverageReporters": [
       "text-summary"
+    ],
+    "collectCoverageFrom": [
```

Comment:
I added this configuration of `jest` because `jest` threw a syntax error on new `tests/fixtures/basics/solo-const.src.js` file.